docs/components/typography/test/index.html

```diff
@@ -0,0 +1,299 @@
+{% extends 'test.njk' %} {% block content %}
+<section>
+    <!-- Caption -->
+    <div class="demo" data-dom-diff="typography-caption">
+        <p class="hxCaption">Caption</p>
+    </div>
+</section>
+<section>
+    <!-- Error Message -->
+    <div class="demo" data-dom-diff="typography-error-message">
```

Comment:
Would it make more sense to set the value of `data-dom-diff` to `hx-error` to align with the name of the custom element we're testing?